tests/std/tests/P0896R4_views_transform/test.cpp

```diff
@@ -577,27 +587,54 @@ struct iterator_instantiator {
             const auto first = r.begin();
             const auto last  = r.end();
 
+            const auto const_first = ranges::iterator_t<const R>{first};
+            const auto const_last  = ranges::sentinel_t<const R>{last};
+
             assert(first == first);
             assert(I{} == I{});
             STATIC_ASSERT(noexcept(first == first));
 
+            assert(first == const_first);
+            assert(const_first == first);
+            STATIC_ASSERT(noexcept(first == const_first));

```

Comment:
It looks like we should have a second `static_assert` here (and the order should be adjusted) because the types aren't the same.